repo.config

```diff
@@ -1,7 +1,80 @@
 REPO_USER=foo
 REPO_PASSWORD=foo
 ENVIRONMENT_MANAGER_NODE=some_host_or_IP
-# the machine fully qualified domain name
-MACHINE_FQDN=osparc.local
+
+# this will set the user/password credentials on all Ops services for a local deployement
 SERVICES_USER=admin
 SERVICES_PASSWORD=adminadmin
+# the machine fully qualified domain name
+MACHINE_FQDN=osparc.local
+MONITORING_DOMAIN=monitoring.${MACHINE_FQDN}
+PORTAINER_DOMAIN=portainer.${MACHINE_FQDN}
+STORAGE_DOMAIN=storage.${MACHINE_FQDN}
+REGISTRY_DOMAIN=registry.${MACHINE_FQDN}
+
+
+# define the stack name
+PREFIX_STACK_NAME=local
+# defines the simcore stack name
+SIMCORE_STACK_NAME=simcore
+# set the image tag to be used from dockerhub
+SIMCORE_IMAGE_TAG=master-latest
+
+# S3
+ACCESS_KEY_ID=${SERVICES_PASSWORD}
+SECRET_ACCESS_KEY=${SERVICES_PASSWORD}
+S3_ENDPOINT=${STORAGE_DOMAIN}
+S3_BUCKET=${PREFIX_STACK_NAME}-simcore
+S3_SECURE=1
+
+# Graylog
+GRAYLOG_LOGIN=${SERVICES_USER}
+GRAYLOG_ROOT_PASSWORD=${SERVICES_PASSWORD}
+
+
+# Postgres
+POSTGRES_DB=simcoredb
+POSTGRES_ENDPOINT=${PREFIX_STACK_NAME}_postgres:5432
+POSTGRES_HOST=${PREFIX_STACK_NAME}_postgres
+
+POSTGRES_USER=${SERVICES_USER}
+POSTGRES_PASSWORD=${SERVICES_PASSWORD}
+POSTGRES_PORT=5432
+
+# registry
+REGISTRY_USER=${SERVICES_USER}
+REGISTRY_PW=${SERVICES_PASSWORD}
+REGISTRY_AUTH=True
+REGISTRY_SSL=True
+REGISTRY_S3_ACCESS_KEY_ID=${ACCESS_KEY_ID}
+REGISTRY_S3_SECRET_ACCESS_KEY=${SECRET_ACCESS_KEY}
+REGISTRY_S3_BUCKET=registry.osparc.io
```

Comment:
this should be:
REGISTRY_S3_BUCKET=${REGISTRY_DOMAIN}

Reply:
From what I remind. the name is always registry.osparc.io, so not always ${REGISTRY_DOMAIN} no ?

Reply:
so I just double-checked on master. there it is osparc01.speag.com for example.
it would not really make sense to always name it with osparc.io. for example in aws the registry is with simcore.io right?